src/content/docs/dev/send-tokens/sui-its.mdx

```diff
@@ -0,0 +1,502 @@
+        coin.into_balance(),
+        source_channel.to_address(),
+        destination_chain,
+        destination_address,
+        metadata,
+        VERSION,
+    )
+}
+```
+
+#### Send Interchain Transfer
+
+This will trigger the cross-chain call to actually send the coin from the source chain to the destination chain.
+
+The function takes three parameters:
+1. `its`: The ITS module that will be updated once the new coin is registered
+1. `ticket`: The [ticket](interchain-transfer-ticket) representing the coin transfer
+1. `clock`: A [clock](https://docs.sui.io/references/framework/sui/clock) module that provides the time of the transfer
+
+```move
+public fun send_interchain_transfer<T>(
+    self: &mut ITS,
+    ticket: InterchainTransferTicket<T>,
+    clock: &Clock,
+): MessageTicket {
+    let value = self.value_mut!(b"send_interchain_transfer");
+
+    value.send_interchain_transfer<T>(
+        ticket,
+        VERSION,
+        clock,
+    )
+}
+```
+
+See here for an [example](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/sui/its-example.js#L65) of how to use the `send_interchain_transfer()` 
+
+The full `send_interchain_transfer()` implementation can be found [here](https://github.com/axelarnetwork/axelar-cgp-sui/blob/main/move/interchain_token_service/sources/versioned/interchain_token_service_v0.move#L212) 
+
+
+### Receive Interchain Transfer
+
+When tokens are sent to Sui, the [Relayer Discovery](https://github.com/axelarnetwork/axelar-cgp-sui/blob/main/move/interchain_token_service/sources/discovery.move) module will trigger the `receive_interchain_transfer()` function on the Sui ITS module. 
+
+The function takes 4 parameters: 
+1. `its`: The ITS module that will be updated once the new coin is registered
+1. `ticket`: The [ticket](interchain-transfer-ticket) representing the token transfer
+1. `clock`: A [clock](https://docs.sui.io/references/framework/sui/clock) module that provides the time of the transfer
+1. `ctx`: The [transaction context](https://docs.sui.io/references/framework/sui/tx_context#sui_tx_context_TxContext) provides the necessary runtime environment for creating or modifying objects and state
+
+```move
+public fun receive_interchain_transfer<T>(
+    self: &mut ITS,
+    approved_message: ApprovedMessage,
+    clock: &Clock,
+    ctx: &mut TxContext,
+) {
+    let value = self.value_mut!(b"receive_interchain_transfer");
+
+    value.receive_interchain_transfer<T>(approved_message, clock, ctx);
+}
+```
+
+Once the relayer triggers this function it in turn triggers the [give_coin()](give-coin) function on the Coin Management program. Then once `give_coin()` has run the function will transfer the coin to the destination address.
+
+The full `receive_interchain_transfer()`  can be found [here](https://github.com/axelarnetwork/axelar-cgp-sui/blob/main/move/interchain_token_service/sources/versioned/interchain_token_service_v0.move#L247) 
+
+
+#### Receive Interchain Transfer With Data
```

Comment:
Please mention that if an app wishes to receive tokens with data they need to register themselves with `relayer_discovery` and use a channel they control as the destination address. The implementation of squid in this repo can be used as a reference.